### ISSUE TITLE
hello world 

### ISSUE THREAD

**Issue by abhishekkumkar** [dscvitc/hacktoberfest#1]
How to program hello world in python?

**Comment by SuphalBhattarai** [comment]
make a file named a .py extension
the add the following line 
`print("Hello World")`

**Comment by sfxjolugbohunmi** [comment]
@abhishekkumkar Can I contribute by writing a hello world program in Kotlin?
If yes, I can open a PR soon

**Comment by abhishekkumkar** [comment]
@sfxjolugbohunmi Yes you can contribute. Create a pr request. We will review it and then merge it.

**Comment by abhishekkumkar** [comment]
@SuphalBhattarai thank you for your help.

**Comment by mayorJAY** [comment]
@abhishekkumkar Thanks 
This is still sfxjolugbohunmi
I'll be creating the PR with this my personal GitHub account

**Comment by sid-1207** [comment]
Can I solve this issue in C?

**Comment by rishika-singh49** [comment]
can you please assign this to me?

**Comment by abhishekkumkar** [comment]
@rishika-singh49 It's open for all. You can send your solution too.

**Comment by sid-1207** [comment]
Solution in C/C++ is allowed right?

**Comment by abhishekkumkar** [comment]
@sid-1207 Yes

**Comment by normal-crayon** [comment]
i hope haskell helloworld is contribute-able. PR given.
